changelogs/unreleased/0000000001-typo_fix.yml

```diff
@@ -0,0 +1,9 @@
+---
+description: Fixed two typos in language.rst file of core docs
+change-type: patch
+destination-branches: [master, iso3, iso4]
+sections:
+  bugfix: "Fixed typos in language.rst file"
+  other-note: |
```

Comment:
the other-note I would remove as well, this is not sufficiently important to put in the changelog.